app/src/docs/content/tutorials/buildingPubSub.mdx

```diff
@@ -18,40 +18,30 @@ import docsStyles from '$docs/components/DocsLayout/docsLayout.pcss'
 import tutorialStyles from '$docs/components/Pages/Tutorials/tutorials.pcss'
 
 # Building a simple pub/sub system
-Before proceeding with this tutorial, we assume you have <Link to={docsLinks.gettingStarted}>got your API key or connected your Ethereum account</Link> and that you have also <Link to={docsLinks.streams}>created your first stream</Link>.
+Before proceeding with this tutorial, we assume you have connected your Ethereum account and that you have also <Link to={docsLinks.streams}>created your first stream</Link>.
 
 ### Publish to a stream
-The easiest way to push data to a stream is to use the one of our <Link to={docsLinks.sdk}>SDKs</Link>, currently available in <a target="_blank" href="https://github.com/streamr-dev/streamr-client-javascript">Javascript</a>, <a target="_blank" href="https://github.com/streamr-dev/streamr-client-java">Java</a> and a community contributed version in <a target="_blank" href="https://github.com/streamr-dev/streamr-client-python">Python</a>. Client libraries for other languages are on the roadmap. Here is an example of publishing data to a stream using Streamr Javascript client.
+The easiest way to push data to a stream is to use one of our <Link to={docsLinks.sdk}>SDKs</Link>, currently available in <a target="_blank" href="https://github.com/streamr-dev/streamr-client-javascript">Javascript</a>, <a target="_blank" href="https://github.com/streamr-dev/streamr-client-java">Java</a>. Client libraries for other languages are on the roadmap. Here is an example of publishing data to a stream using Streamr Javascript client.
 
 <CodeSnippet language="javascript" wrapLines showLineNumbers >{ClientPub}</CodeSnippet>
 
-You can also try it locally in your browser. Just open this <a target="_blank" href="https://jsbin.com/xahoxut/edit?js,output">JS Bin</a> or <a target="_blank" href="https://gist.github.com/fonty1/f09aa09afcbae0bf5d7032163a466f76">GitHub Gist</a>, and replace `MY-STREAM-ID` and `MY-API-KEY` with your stream ID and API key. Your browser will start producing data to your stream! If you were successful, you should see something like this in your stream editor preview.
-
 <Image
   src={StreamEditorPreview}
   highResSrc={StreamEditorPreview2x}
   alt="Stream Editor Preview"
 />
 
-Additionally, you can interact with the Streamr API using any HTTP library of your choice. You will find all the details in the <Link to={docsLinks.api}>API section</Link>, but here’s a brief rundown. You’ll be making a `HTTP POST` request to a URL, which contains your stream ID.
+Additionally, you can interact with the Streamr API using any HTTP library of your choice. You will find all the details in the <Link to={docsLinks.api}>API section</Link>, but here’s a brief rundown. You’ll be making a `HTTP POST` request to a URL, which contains your stream ID. Please note that slashes in the stream ID should be escaped using `%2F`.
```

Comment:
👍

Reply:
Maybe "...should be escaped using url encoding such as changing slashes to `%2F`" or something like that because not only slashes should be encoded: https://en.wikipedia.org/wiki/Percent-encoding